package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@cumulus/cumulus-message-adapter-js",
-  "version": "0.0.1",
+  "version": "0.0.1-beta.2",
```

Comment:
I think beta.3 is the current version and this should be updated in package.json, not package-lock
https://www.npmjs.com/package/@cumulus/cumulus-message-adapter-js

Reply:
merge in master to get the latest version